static/scripts/loggedin.js

```diff
@@ -9,6 +9,12 @@ if (window.opener && window.opener !== window) {
 
 function toggleMobileNav() {
     document.querySelector('aside.nav-sidebar').classList.toggle('active');
+	const announcementAlert = document.querySelector('.alert-announcement');
+	if (announcementAlert.style.display === 'none') {
+		announcementAlert.style.display = 'block';
+	} else {
+		announcementAlert.style.display = 'none';
```

Comment:
I dont get that if clause. If its "none" put it in "block" but if its anything else, than put it to "none" ? could you explain that somehow?

Reply:
this is because we ant to hide this announcement when we click menu button (it breaks styling). So if announcement was hidden after clicking menu button we want to show it (it means that we are closing navigation menu) and if it was displayed and we click menu button we want to hide it as it means that we open navigation menu